physics/SFC_Models/Land/Noahmp/module_sf_noahmplsm.F90

```diff
@@ -1989,7 +1989,7 @@ subroutine energy (parameters,ice    ,vegtyp ,ist    ,nsnow  ,nsoil  , & !in
 
   real (kind=kind_phys), parameter                   :: mpe    = 1.e-6
   real (kind=kind_phys), parameter                   :: psiwlt = -150.  !metric potential for wilting point (m)
-  real (kind=kind_phys), parameter                   :: z0     = 0.002  ! bare-soil roughness length (m) (i.e., under the canopy)
+  real (kind=kind_phys), parameter                   :: z0     = 0.015  ! bare-soil roughness length (m) (i.e., under the canopy)
```

Comment:
where does "0.015" come from? Is it tuned? I did not find this value in the Noah-MP.

Reply:
it is tuned based on wind speed bias over bare soil and vegetated areas with low vegetation fraction

Reply:
got it. thanks!